cube/swiss/source/config/config.c

```diff
@@ -51,8 +51,7 @@ static int configEntriesCount = 0;
 
 
 void strnscpy(char *s1, char *s2, int num) {
-	strncpy(s1, s2, num);
-	s1[num] = 0;
+	sprintf(s1, "%.*s", num, s2);
```

Comment:
We have strlcpy. Shouldn't this be used here?

Reply:
You're right.